pkg/systemd/terminal.html

```diff
@@ -35,9 +51,12 @@
         term.open(container[0]);
         container.children().first().css('margin', 0);
 
+        var user_shell = "/bin/bash";
+        if (current_user.shell !== undefined)
```

Comment:
> I think you can just use if (current.user.shell)

This could (technically) be a problem if `current.user.shell` interpreted as a boolean value evaluates to false. Shouldn't happen in this case, but I consider it bad form.